src/style.css

```
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.block {
+  height: 300px;
+  text-align: center;
+  line-height: 300px;
+  min-width: 33.33%;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #ffff;
+}
+
+.block_1 {
+  background-color: rgb(255, 0, 0);
+  order: 1;
+  flex-grow: 1;
```

```suggestion
  flex-grow: 1;
```
не потрібно дублювати це для кожного блоку. Можеш прописати цю властивість для загального класу .block